fastlane/swift/LaneFileProtocol.swift

```diff
@@ -39,6 +39,10 @@ open class LaneFile: NSObject, LaneFileProtocol {
     private static func trimLaneWithOptionsFromName(laneName: String) -> String {
         return String(laneName.prefix(laneName.count - 12))
     }
+  
+    public func onError(currentLane: String, errorInfo: String, errorClass: String?, errorMessage: String?) {
+      LaneFile.onErrorCalled[currentLane] = true
```

Comment:
If we decided to use a `Set` for example this will become:

```suggestion
      LaneFile.onErrorCalled.insert(currentLane)
```